xtos/include/sof/lib/perf_cnt.h

```diff
@@ -17,8 +17,8 @@
 #include <rtos/timer.h>
 
 struct perf_cnt_data {
-	uint32_t plat_ts;
-	uint32_t cpu_ts;
+	uint64_t plat_ts;
+	uint64_t cpu_ts;
```

Comment:
You must explain real code changes like this one.

The commit message sounds like it's merely adding a new build configuration but this does more than that.

This most likely requires multiple commits https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes

Reply:
CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER this was enabled already, current cpu_ts and plat_ts are all get from:
k_cycle_get_64

32 bit need more implementation to avoid overflow,  it is just for profiling and only be enabled with -o app/perf_overlay.conf.